templates/info.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Linley/Pitts Wedding Info</title>
+    <link rel="stylesheet" href="styles/main.css" type="text/css"/>
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script|Lato|Lora" rel="stylesheet">
+  </head>
+  <body>
+    <section class="sitemap-header">
+      <span class="sitemap-name">David Linley</span>
+      <div class="link-container">
+        <a href="index" class="sitemap-link">RSVP</a>
```

Comment:
This should probably go to `/` rather than `index`, and the href for the info page should also change to `info`.